app/utils/url.js

```diff
@@ -43,6 +43,26 @@ export function isYoutubeLink(link) {
     return link.trim().match(ytRegex);
 }
 
+export async function getShortenedImageLink(link) {
+    return new Promise(((resolve, reject) => {
+        const xhr = new XMLHttpRequest();
```

Comment:
One problem that I have with this is that in case the app is configured to use Mutual TLS the client certificate is not being used and the request will fail

Reply:
n/a as discussed.